File: programming languages/5_physics_problems_in_7_programming_languages/Задачи VB/WindowsApplication1/Form3.vb
Public Class Form3
    Dim t, S, v As Double
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        S = Val(textBox2.Text)
        v = Val(textBox3.Text)

        t = S / v

        textBox1.Text = CStr(t)
    End Sub

    Private Sub Form3_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub
End Class

File: programming languages/5_physics_problems_in_7_programming_languages/Задачи C#/Задачи/Form3.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Задачи
{
    public partial class Form3 : Form
    {
        public Form3()
        {
            InitializeComponent();
        }
        Double t, S, v;

        private void Form3_Load(object sender, EventArgs e)
        {

        }

        private void label4_Click(object sender, EventArgs e)
        {

        }

        private void button1_Click(object sender, EventArgs e)
        {
            S = Convert.ToDouble(textBox2.Text);
            v = Convert.ToDouble(textBox3.Text);
            
            t = S / v;
            textBox1.Text = Convert.ToString(t);
        }
    }
}
